src/BlazorUI/Bit.BlazorUI/Styles/Fluent/shadows.fluent-light.scss

```
@@ -1,3 +1,17 @@
+// Fluent - light elevation. Every value below is Fabric's own depth ramp: a KEY shadow that gives
+// the surface its height and a tighter AMBIENT one that seats it, both generated from the depth
+// number N as
+//   key     0 (N * 0.4)px   (N * 0.9)px   0 colorNeutralShadowKey
+//   ambient 0 (N * 0.075)px (N * 0.225)px 0 colorNeutralShadowAmbient
+// which reproduces the published depth4 / depth8 / depth16 / depth64 exactly and continues the ramp
+// at 2 and 32 where the library needs steps Fabric never named. The alphas are Fabric's .132 key
+// and .108 ambient; depth64 steps them up to .22 / .18, as Fabric does, because a surface that far
+// off the page has to carry the separation on its own.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)